### PR TITLE
Improve the README

### LINKED ISSUE #1183: Outdated README.md
The translation section is outdated.
1. configuring `url` in `_config.yml` is not needed anymore.
2. language config like `section`, `description` has been extracted to `_data/locales/[language_code].yml` not in `_config.yml` anymore

If such PR is welcome. I'm happy to help :)

### PR DESCRIPTION
**Pull request by doomspork**
Closes #1183

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,45 +31,24 @@ In addition to the steps above there are a few addition steps required for trans
 
 #### New Language
 
-1. Create a folder using the 2 character code (e.g. ja, en, es, etc) with lesson subfolders:
+1. Create a folder using the ISO language code (e.g. ja, zh-hans, es, et al) with lesson subfolders:
```
Comment (eksperimental) [inline README.md]: we need to link to this page, and ask people to find the valid 2 letter, or 3 letter code.
https://www.loc.gov/standards/iso639-2/php/English_list.php

File: README.md [hunk 2]
```diff
@@ -39,45 +31,24 @@ In addition to the steps above there are a few addition steps required for trans
...
-  ```
-
-1. Add it to list in `index.md`:
-
-  ```markdown
-  Available in [Việt ngữ][vi], [汉语][cn], [Español][es], [Slovenčina][sk], [日本語][ja], [Polski][pl] [Português][pt], [Русском][ru] and [Bahasa Melayu][ms] and other.
+  script_direction: rtl
   ```
 
 #### Translated Lesson
```
Comment (eksperimental) [inline README.md]: versioning instructions just wrong.

>     * `version` should consist of three digits: `major.minor.patch`, so:
>       * if this is a initial lesson translation, the version should be set to `1.0.0`;
>       * if you apply the original lesson updates to the translation, the version should be copied from the corresponding state of the original lesson;
>       * else bump one of the version numbers depending on how important is your change.

the version is given by the original English version. if they make changes or corrections, still they cannot bump the version number, and the version starts at whatever version the English version is at, not at 1.0.0
that applies for English lessons only

Comment (eksperimental) [inline README.md]: we also need to add that they need to add their language to `_data/interlang.yml` , create a new list, and also add the language item to every list there

Comment (eksperimental) [reply README.md]: If it's a new lesson, add lesson to `_data/contents.yml`

Comment (doomspork) [reply README.md]: There's so many more steps now!  😓

**Pull request merged**